<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsCustomProjectionDialogBase</class>
 <widget class="QDialog" name="QgsCustomProjectionDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Custom Projection Definition</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>10</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget2" >
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Define</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel5" >
         <property name="font" >
          <font>
           <family></family>
           <pointsize>18</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text" >
          <string>Custom Projection Definition</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="leParameters" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="textLabel3_2" >
         <property name="text" >
          <string>Parameters:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2" >
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="pbnFirst" >
           <property name="text" >
            <string>|&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbnPrevious" >
           <property name="text" >
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblRecordNo" >
           <property name="text" >
            <string>1 of 1</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbnNext" >
           <property name="text" >
            <string>></string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbnLast" >
           <property name="text" >
            <string>>|</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbnNew" >
           <property name="text" >
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbnSave" >
           <property name="text" >
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbnDelete" >
           <property name="text" >
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbnClose" >
           <property name="text" >
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="leName" />
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="Q3TextBrowser" name="textBrowser1_2" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Test</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" colspan="4" >
        <widget class="QLabel" name="textLabel1_2" >
         <property name="text" >
          <string>Transform from WGS84 to the chosen projection</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="4" >
        <widget class="QPushButton" name="pbnCalculate" >
         <property name="text" >
          <string>Calculate</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3" >
        <widget class="QLabel" name="textLabel2_3" >
         <property name="text" >
          <string>Projected Corrdinate System</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3" >
        <widget class="QLabel" name="projectedX" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="frameShape" >
          <enum>QFrame::Box</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QLabel" name="textLabel1_3" >
         <property name="text" >
          <string>Geographic / WGS84</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" >
        <widget class="QLineEdit" name="northWGS84" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_2" >
         <property name="text" >
          <string>North:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2" >
        <widget class="QLabel" name="textLabel2_2_2" >
         <property name="text" >
          <string>East:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" >
        <widget class="QLineEdit" name="eastWGS84" />
       </item>
       <item row="5" column="3" >
        <widget class="QLabel" name="projectedY" >
         <property name="enabled" >
          <bool>true</bool>
         </property>
         <property name="frameShape" >
          <enum>QFrame::Box</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="margin" >
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>41</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="4" >
        <widget class="Q3TextBrowser" name="textBrowser1" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="textLabel3_2_2" >
         <property name="text" >
          <string>Parameters:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3" >
        <widget class="QLineEdit" name="leTestParameters" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>QWidget</extends>
   <header>q3textbrowser.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>leParameters</tabstop>
  <tabstop>pbnFirst</tabstop>
  <tabstop>pbnPrevious</tabstop>
  <tabstop>pbnNext</tabstop>
  <tabstop>pbnLast</tabstop>
  <tabstop>pbnNew</tabstop>
  <tabstop>pbnSave</tabstop>
  <tabstop>pbnDelete</tabstop>
  <tabstop>pbnClose</tabstop>
  <tabstop>leTestParameters</tabstop>
  <tabstop>northWGS84</tabstop>
  <tabstop>eastWGS84</tabstop>
  <tabstop>pbnCalculate</tabstop>
  <tabstop>tabWidget2</tabstop>
  <tabstop>textBrowser1_2</tabstop>
  <tabstop>textBrowser1</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
